subprojects/docs/src/docs/userguide/migration/upgrading_version_7.adoc

```diff
@@ -193,7 +193,7 @@ systemProp.org.gradle.kotlin.dsl.precompiled.accessors.strict=true
 
 Along with the Java language version, the <<toolchains#toolchains, Java toolchain>> DSL allows configuring other criteria such as specific vendors or VM implementations.
 Starting with Gradle 7.6, toolchain specifications that configure other properties without specifying the language version are considered _invalid_.
-Invalid specifications are deprecated and will become build errors in Gradle 8.0.
+Usage of invalid specifications results in build errors since Gradle 8.0.
```

Comment:
I do not think this entry should be updated.
Instead you should add a new entry to the 8.0 items "Warnings that are now errors"

Reply:
Done